fuel-vm/src/tests/metadata.rs

```diff
@@ -622,6 +624,12 @@ fn get_transaction_fields() {
         op::eq(0x10, 0x10, 0x11),
         op::and(0x20, 0x20, 0x10),
 
+        op::movi(0x19, 0x00),
+        op::movi(0x11, owner_idx as Immediate18),
```

Comment:
question for me to understand: why `Immediate18`? By looking at the specs and the [implementation](https://github.com/FuelLabs/fuel-vm/blob/9079849648355542a9a3adb80ed5d62a3abe283b/fuel-asm/src/op.rs#L38) of `GTF`, I would expect `Immediate12` here?

Reply:
I don't know I just followed the existing pattern. Maybe @Dentosal knows